### DISCUSSION TITLE
Disable per-memory TX?

### DISCUSSION THREAD

**Discussion by oisact** [egzumer/uv-k5-firmware-custom#238] [Q&A]
I looked through the manual and settings, and don't see any obvious way to disable TX per-channel. Is this possible? IE I have a public service channel I don't want to accidentally TX on. Thanks.

**Comment by ErikS-web** [comment]
What frequency ?

**Reply by oisact** [reply]
Just any arbitrary memory channel I choose. In other radios with per-channel TX power levels there is often a DISABLE setting that does not allow transmitting for specific channels, in addition to the usual LO, HI, etc power settings.

**Comment by f5gkw** [comment]
By set in tx a " bad " frequency maybe ? And a "good" one in Rx field
Or try a very large offset value.

**Comment by Hunterfnet** [comment]
It's so easy as change and save the channel on memory with AM/USB mode and you can't transmit nothing on this channel .
